### PR TITLE
customized invitation

### PR DESCRIPTION
**Pull request by seanmarcia**
This goes along with the `customized_invitation` branch on the diaper side.

This PR allows diaper banks to add in their own custom messages when inviting partners to partnerbase.

### CODE REVIEW

File: app/views/devise/mailer/invitation_instructions.html.erb [hunk 1]
```diff
@@ -341,10 +341,13 @@
...
                   <td>
-                    <p>Hi there!</p>
-
+                    <p>Hello <%= @resource.email %>!</p>
+                    <% if @resource.message %>
+                      <p><%= @resource.message %></p>
+                    <% else %>
                     <p>You've been invited to become a partner organization with a diaper bank or menstrual supply organization!</p>
                     <p>Please click the link below to accept your invitation and create an account and you'll be able to begin requesting distributions.</p>
+                      <% end %>
```
Comment (benreyn) [inline app/views/devise/mailer/invitation_instructions.html.erb]: Indentation feels a little off here.

**Comment by benreyn** [comment]
One final question. Should we add a tests to cover this use case, or do you feel like devise covers this well enough for us?

**Comment by seanmarcia** [comment]
Yeah, that's something I struggle with since it is `devise-invitable` functionality. I updated the tests on the diaper side but since this is essentially testing the gem I've always felt like the the gem's own tests should be good enough -- not that I've ever looked at or run the test suite on a gem :)

**Comment by benreyn** [comment]
Sounds good to me. Lets merge this thing.

**Pull request merged**